src/geo/map-view.js

```diff
@@ -192,7 +212,19 @@ var MapView = View.extend({
 
 
 }, {
-  _getClass: function(provider) {
+  
+  /**
+   * Factory method that given a {@link MapModel} instance with visualization
+   * configuration creates a new concrete view instance depending on the provider.
+   *
+   * @param {HTMLElement} el HTML element where to attach the map.
+   * @param  {MapModel} mapModel MapModel instance
+   * @return {(LeafletMapView|GoogleMapsMapView)}  New MapView's subclass instance.
+   */
+  create: function(el, mapModel) {
+
+    // Choose a concrete subclass implementation
+    var provider = mapModel.get('provider');
     var mapViewClass = cdb.geo.LeafletMapView;
     if(provider === 'googlemaps') {
```

Comment:
This is something that could probably be extracted from here. I guess we could check this when we parse the viz.json. If the provider is googlemaps and google is not defined, it doesn't make sense to get to this point. Right?